assets/js/components/notifications/ErrorNotifications.test.js

```diff
@@ -90,4 +90,21 @@ describe( 'ErrorNotifications', () => {
 			} )
 		);
 	} );
+
+	it( 'renders GTE message for unsatisfied tagmanager readonly scope', () => {
+		provideUserAuthentication( registry, {
+			unsatisfiedScopes: [
+				'https://www.googleapis.com/auth/tagmanager.readonly',
+			],
+		} );
+
+		const { container } = render( <ErrorNotifications />, {
+			registry,
+			features: [ 'gteSupport' ],
+		} );
+
+		expect( container ).toHaveTextContent(
+			'Site Kit needs additional permissions to detect updates to tags on your site'
+		);
```

Comment:
Rather than just testing the title here, this seems like an appropriate place to use a snapshot test, which would verify the rest of the rendered output (the main message, etc).

Reply:
Thanks, @jimmymadon, that's a good addition. However, I was referring to Jest snapshot tests, i.e. using `toMatchSnapshot()`. Arguably, the same applies to other tests within the suite, as they are a bit light on coverage in that they only test the title with no coverage for the message.

What do you think, we could add snapshots or maybe some more assertions?